src/main.rs

```diff
@@ -800,16 +800,20 @@ impl Cli {
                     .set_title("Uninstall")
                     .set_text("Delete Space Acres configuration and logs for all users?")
                     .show_confirm()
-                    .is_ok_and(|x| x)
+                    .unwrap_or_default()
             {
                 if let Some(system_drive) = std::env::var_os("SystemDrive") {
                     // Workaround for https://github.com/rust-lang/rust-clippy/issues/12244
-                    #[allow(clippy::all)]
+                    #[allow(clippy::join_absolute_paths)]
                     let users_dir = std::path::PathBuf::from(system_drive).join("\\Users");
```

Comment:
Are you sure this doesn't work (in which case clippy suppression is not necessary)?:
```suggestion
                    let users_dir = PathBuf::from(system_drive).join("Users");
```

Reply:
How fun :disappointed: